### DISCUSSION TITLE
Let's confirm you are human issue

### DISCUSSION THREAD

**Discussion by forzaturian** [cypress-io/cypress#29260] [Questions and Help]
I had no issue running Cypress tests, but today I've faced an identification issue on all browsers. 
Even after passing a picture puzzle manually, I'm getting an incorrect password error while login.

<img width="757" alt="Screenshot 2024-04-03 at 1 39 06 PM" src="https://github.com/cypress-io/cypress/assets/47934778/2eb6d32c-8bfb-44fb-80d2-f2d5d657b681">
<img width="760" alt="Screenshot 2024-04-03 at 1 40 35 PM" src="https://github.com/cypress-io/cypress/assets/47934778/9012e8d4-dea9-4771-9262-ef57a6ce7950">
<img width="439" alt="Screenshot 2024-04-03 at 1 41 03 PM" src="https://github.com/cypress-io/cypress/assets/47934778/1fd302bd-4a6f-48af-a466-1eda153d88bf">
<img width="754" alt="Screenshot 2024-04-03 at 1 57 20 PM" src="https://github.com/cypress-io/cypress/assets/47934778/dd42e7d0-3cd1-4d6d-8752-4611f94147f1">

**Comment by portyu9** [comment]
Hi @forzaturian 

This looks like bot/risk detection from the site rather than a Cypress browser issue.

The key clue is that the CAPTCHA now appears in **all Cypress browsers**, while the same login works normally outside Cypress. Solving the picture puzzle manually inside a Cypress-controlled session does not necessarily make that session trusted—the site's anti-bot system may use additional signals and still reject the authentication attempt. The "incorrect password" message may therefore be a generic login rejection rather than proof that Cypress entered the password incorrectly.

I would not try to bypass the CAPTCHA by changing browser fingerprints, user agents, or automation flags.

If this is an application you control, the normal testing approach is to:

* disable CAPTCHA or use the CAPTCHA provider's official test mode/keys in the test environment,
* test the CAPTCHA integration separately, and
* authenticate most Cypress tests programmatically with `cy.request()` and cache the session with `cy.session()`.

For example:

```js
cy.session('test-user', () => {
  cy.request('POST', '/api/login', {
    username: Cypress.env('username'),
    password: Cypress.env('password'),
  })
})
```

If this is a third-party site that you do not control, automating its login UI is inherently unreliable and may violate its automation policy. Use its supported API/OAuth flow instead if one is available.

So I wouldn't troubleshoot this as a selector, timeout, or Chrome-vs-Firefox problem—the evidence points to the authentication provider/site deliberately challenging the automated session.
